logging/fields/fields.go

```diff
@@ -269,3 +269,7 @@ func Errors(val []error) zap.Field {
 func ToBlock(val *big.Int) zap.Field {
 	return zap.Int64(FieldToBlock, val.Int64())
 }
+
+func FormatDutyID(epoch phase0.Epoch, duty *spectypes.Duty) string {
```

Comment:
somewhere on our way we missed these changes https://github.com/bloxapp/ssv/pull/903
I think we need to pull them and them refactor code in current branch regarding top changes we lost.